### ISSUE TITLE
EncodingWarning in faulthandler

### ISSUE THREAD

**Issue by jaraco** [pytest-dev/pytest#11019]
With PYTHONWARNDEFAULTENCODING, pytest emits a warning on startup:

```
 jaraco.abode main $ env PYTHONWARNDEFAULTENCODING=1 .tox/py/bin/python -m pytest
/Users/jaraco/code/jaraco/jaraco.abode/.tox/py/lib/python3.11/site-packages/_pytest/faulthandler.py:30: EncodingWarning: 'encoding' argument not specified
  config.stash[fault_handler_stderr_key] = open(stderr_fd_copy, "w")
...
```

**Comment by jaraco** [comment]
Looks like it was fixed in #10954.

**Comment by RonnyPfannschmidt** [comment]
Thanks for the follow up
